.github/workflows/test_mupdf-release-branch.yml

```diff
@@ -1,7 +1,6 @@
 name: Test mupdf release branch
 
 on:
-  
```

Comment:
Please revert unnecessary blank lines removals/additions.

Reply:
I have backed out pure-whitespaces changes, rebased and force-pushed.